web/vue/src/components/gekko/singleStratrunner.vue

```diff
@@ -84,6 +84,7 @@ import { post } from '../../tools/ajax'
 import spinner from '../global/blockSpinner.vue'
 import chart from '../backtester/result/chartWrapper.vue'
 import paperTradeSummary from '../global/paperTradeSummary.vue'
+import config from '../../../../routes/baseConfig.js'
```

Comment:
You don't actually need to include it here since in the route the we already merge whatever is send over by the frontend: https://github.com/askmike/gekko/blob/stable/web/routes/startGekko.js#L19